testing/deployer/sprawl/catalog.go

```diff
@@ -723,7 +725,7 @@ func workloadInstanceToResources(
 			Data: &pbcatalog.Workload{
 				// TODO(rb): disabling this until node scoping makes sense again
 				// NodeName: node.PodName(),
-				Identity: wrk.ID.Name,
```

Comment:
bug3: We disregarded the user-specified `WorkloadIdentity` field when creating the workload.